source/_integrations/nuki.markdown

```diff
@@ -8,13 +8,17 @@ ha_iot_class: Local Polling
 ha_codeowners:
   - '@pschmitt'
   - '@pvizeli'
+  - '@pree'
 ha_domain: nuki
 ---
 
 The `nuki` platform allows you to control [Nuki Smart Locks](https://nuki.io/en/smart-lock/) via either a [software bridge](https://play.google.com/store/apps/details?id=io.nuki.bridge) or a [physical bridge](https://nuki.io/en/bridge/).
 
 To add a Nuki bridge to your installation, you need to enable developer mode on your bridge and define a port and an access token. This can be achieved using the [Android app](https://play.google.com/store/apps/details?id=io.nuki) or [iPhone app](https://apps.apple.com/app/nuki-smart-lock/id1044998081). Go to manage my devices, and select the bridge. Within the bridge configuration turn on the HTTP API and check the details in the screen. Please note that the API token should be 6-20 characters long, even though the app allows you to set a longer one.
-Then add the following to your `configuration.yaml` file:
+
+Once you have the token set, to add `Nuki` to your installation, go to **Configuration** >> **Integrations** in the UI, click the button with `+` sign and from the list of integrations select **Nuki**.
+
+Alternatively, add the following to your `configuration.yaml` file:
```

Comment:
The YAML configuration should have been deprecated and thus removed, right?

Reply:
Yes it's now deprecated, but still used for import. I removed it from the documentation